pkg/replication/replicator.go

```diff
@@ -134,14 +134,14 @@ func (r replicator) initializeReplicationWithCheck(
 	overrideLock bool,
 	ignoreControllers bool,
 	concurrentRealityRequests int,
-	checkPreparers bool,
+	chekPreparers bool,
```

Comment:
this is when I tried to do a sed to fix my previous sed :D i've learned a lesson now